frontend/src/components/Forms/MineRecordForm.js

```diff
@@ -3,31 +3,64 @@ import PropTypes from "prop-types";
 import { connect } from "react-redux";
 import { compose } from "redux";
 import { Field, reduxForm, FieldArray, getFormValues } from "redux-form";
-import { Form, Button, Col, Row, Popconfirm, Icon, Collapse } from "antd";
+import { Form, Button, Col, Row, Popconfirm, Icon, Collapse, notification, Tag } from "antd";
+import _ from "lodash";
```

Comment:
We're destructuring lodash imports elsewhere. May as well do that here for consistency